src/commands/projects/get.ts

```diff
@@ -1,13 +1,30 @@
-import { fetchProjects } from '../../api/projects'
-import Base from '../base'
+import { GetProjectsParams, fetchProjects } from '../../api/projects'
+import GetCommand from '../getCommand'
 
-export default class DetailedProjects extends Base {
+export default class DetailedProjects extends GetCommand {
     static description = 'Retrieve all projects in the current Organization'
     static hidden = false
     authRequired = true
 
+    static flags = {
+        ...GetCommand.flags,
+    }
+
     public async run(): Promise<void> {
-        const projects = await fetchProjects(this.authToken)
+        const { flags } = await this.parse(DetailedProjects)
+        const { sortBy, sortOrder, headless } = flags
+
+        const params = await this.populateParameters(
```

Comment:
Is calling `populateParameters `still necessary if we're no longer prompting the user? can't the sorting flags be passed directly to `fetchProjects` if they're defined?